jwst/saturation/saturation.py

```diff
@@ -8,7 +8,7 @@
 from . import x_irs2
 
 log = logging.getLogger(__name__)
-log.setLevel(logging.DEBUG)
+log.setLevel(logging.INFO)
```

Comment:
Want to keep this set to DEBUG, so that debug-level messages will be passed when requested.